e2e/runner/cypress-runner-backend.js

```diff
@@ -1,130 +1,70 @@
 #!/usr/bin/env node
 
-const { spawn } = require("child_process");
+const { execSync, spawn } = require("child_process");
 const fs = require("fs");
-const http = require("http");
 const os = require("os");
 const path = require("path");
 
-const { BACKEND_HOST, BACKEND_PORT } = require("./constants/backend-port");
-const { delay } = require("./cypress-runner-utils");
+const { waitUntilReady, shell } = require("./cypress-runner-utils");
 
-const CypressBackend = {
-  server: null,
-  createServer(port = BACKEND_PORT) {
-    const generateTempDbPath = () =>
-      path.join(os.tmpdir(), `metabase-test-${process.pid}.db`);
+const tempDbPath = path.join(os.tmpdir(), `metabase-test-${process.pid}.db`);
+
+function getJvmOptsFromDepsEdn(alias = "e2e") {
+  const cmd = `clojure -Sdeps '{:deps {}}' -M -e '(->> (-> "deps.edn" slurp clojure.edn/read-string :aliases :${alias} :jvm-opts) (clojure.string/join " ") println)'`;
+  return execSync(cmd, { encoding: "utf8" }).trim().toString();
+}
```

Comment:
I am hoping to move the whole E2E setup to `mage` in near future. No more spawning and messing with processes from Node.

Until then, this will do.